docs/eventing/broker/_index.md

```diff
@@ -1,7 +1,27 @@
 ---
-title: "Broker"
+title: "Brokers"
 weight: 60
 type: "docs"
+showlandingtoc: "false"
 ---
 
-{{% readfile file="README.md" %}}
+Brokers are Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that define an event mesh for collecting a pool of [CloudEvents](https://cloudevents.io/), and provide a discoverable endpoint, `status.address`, for event delivery. Event producers can send events to a broker by POSTing the event to the `status.address.url` of the broker.
+
+Event delivery mechanics are an implementation detail that depend on the configured [broker class](./configuring-brokers/#configuring-the-default-broker-class). Using brokers and triggers abstracts the details of event routing from the event producer and event consumer.
+
+<img src="images/broker-workflow.svg" width="70%">
+
+Once an event has entered a broker, it can be forwarded to subscribers by using triggers. Triggers allow events to be filtered by attributes, so that events of a particular class can be sent to subscribers that have registered interest in that class of events.
```

Comment:
`class of events` might sound too formal, like we are defining event classes, where what we really want to say is something like: 

> , so that events with particular attributes can be sent to subscribers that have registered interest in events with those sets of attributes.

?